frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -91,6 +98,30 @@ const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
       namespace,
       prop: 'buildconfigs',
     },
+    {
+      isList: true,
+      kind: referenceForModel(RevisionModel),
```

Comment:
What happens to topology when knative CRDs aren't available?

Reply:
> What happens to topology when knative CRDs aren't available?

+1, this needs a feature flag based on the presence of the CRDs. Then we only try to watch resources that exist. Otherwise it's an error.